src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -140,8 +142,18 @@ struct OperationViewModel {
             responses.append(ResponseViewModel(from: response, with: operation))
         }
 
+        var defaultException: ExceptionResponseViewModel?
         for exception in operation.exceptions ?? [] {
-            exceptions.append(ExceptionResponseViewModel(from: exception))
+            let vm = ExceptionResponseViewModel(from: exception)
+            if vm.isDefaultStatusCode {
+                assert(
+                    defaultException == nil,
+                    "Multiple default exception per operation is currently not supported... \(operation.name)"
+                )
```

Comment:
I don't think there could be multiple defaults anyhow since it is a dictionary and the keys have to be unique, right?

Reply:
yeah I removed the Assert.